src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -205,11 +199,15 @@ public void sendDisabledDatasetsMessage(User user, List<String> disabledDatasets
     public void sendClosedDataSetElectionsMessage(List<Election> elections) throws MessagingException, IOException, TemplateException {
         if(isServiceActive){
             Map<String, List<Election>> reviewedDatasets = new HashMap<>();
+            List<String> referenceIds = elections.stream().map(Election::getReferenceId).collect(Collectors.toList());
+            List<DarCollection> darCollections = collectionDAO.findDARCollectionsByReferenceIds(referenceIds);
             for(Election election: elections) {
                 List<Election> dsElections = electionDAO.findLastElectionsByReferenceIdAndType(election.getReferenceId(), ElectionType.DATA_SET.getValue());
-                DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(election.getReferenceId());
```

Comment:
It looks like `dataAccessRequestDAO` and `userPropertyDAO` are unused in this class, would be nice to remove them.